### DISCUSSION TITLE
Clarify Divergence Loss in UI

### DISCUSSION THREAD

**Discussion by pogoetic** [Uniswap/interface#1687] [Ideas]
Today the Liquidity value is only displayed in $ USD and it conflates two things, change in liquidity value due to change in notional $ value of the assets in the LP and the change in liquidity value due to "impermanent loss" (divergence loss) or the shift in relative price point between the two assets from the midpoint (50/50 price).

Would it be possible to separate these two concepts so LPs had a better sense of their profit/loss due to $ value changes in the two assets, vs. relative price changes in the two assets? Perhaps an option to toggle the liquidity value into ETH or one of the two assets being provided, rather than $ USD - this would remove the notional USD value from the profit equation.

For example, I stake $20,000 in 50/50 ETH/WBTC, let's say. Now my ratio has moved to 60/40 ETH/WBTC and my liquidity value is now $19,500. I can't tell if the $500 notional USD loss came from price movements in ETH or WBTC relative to USD, or due to a price movement between ETH and WBTC. Hope that makes it more clear.

If I could toggle and see the liquidity value in ETH or WBTC then I could remove the USD price movement from the equation and observe the size of my "impermanent loss" (divergence loss). For stablecoin pairs pegged to USD, this feature is not necessary (ex. ETH/DAI). But for all others, it would be mighty helpful.

**Comment by Profka95** [comment]
$20,000
